app/lib/admin/metrics/dimension/software_versions_dimension.rb

```diff
@@ -98,7 +98,10 @@ def imagemagick_version
   end
 
   def ffmpeg_version
-    version = `ffmpeg -version`.match(/ffmpeg version ([\d\.]+)/)[1]
+    version_output = Terrapin::CommandLine.new(Rails.configuration.x.ffmpeg_binary, '-version').run
+    version_match = version_output.match(/ffmpeg version ([\d\.]+|[^\s]+)/)
+
+    version = version_match ? version_match[1] : '0.0.0'
```

Comment:
The following should be future-proof and avoid relying on a specific version format:
```suggestion
    version_output = Terrapin::CommandLine.new(Rails.configuration.x.ffprobe_binary, '-show_program_version -v 0 -of json').run
    version = Oj.load(version_output, mode: :strict, symbol_keys: true).dig(:program_version, :version)
```

Reply:
I didn't realize the ffprobe binary was an env option as well, but makes more sense to do it this way.